### PR TITLE
Chat: Hide current user name

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js [hunk 1]
```diff
@@ -70,6 +70,39 @@ QUnit.module('MessageGroup', moduleConfig, () => {
         });
     });
 
+    QUnit.module('Author name', () => {
+        QUnit.test('value of name element should be equal author name if alignment is start', function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js]:
```suggestion
        QUnit.test('text of a name element should be equal to author name if alignment is start', function(assert) {
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js [hunk 2]
```diff
@@ -70,6 +70,39 @@ QUnit.module('MessageGroup', moduleConfig, () => {
...
+                    },
+                ],
+            });
+
+            const $name = this.$element.find(`.${CHAT_MESSAGE_AUTHOR_NAME_CLASS}`);
+
+            assert.strictEqual($name.text(), name);
+        });
+
+        QUnit.test('value of name element should be equal author name if alignment is end', function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js]:
```suggestion
        QUnit.test('text of a name element should be empty if alignment is end', function(assert) {
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js]: [1d5538fe2ef772e3b09931553ed6fe0b9f41d5a1]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageGroup.tests.js]: [1d5538fe2ef772e3b09931553ed6fe0b9f41d5a1]

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**